client/dashboard/sites/overview-domain-upsell-card/index.tsx

```diff
@@ -69,23 +70,27 @@ const DomainUpsellCardContent = ( {
 		}
 
 		if ( requiresPlanUpgrade( site ) ) {
-			window.location.href = getDomainAndPlanUpsellUrl( {
-				siteSlug: site.slug,
-				backUrl,
-				step: 'plans',
-			} );
+			window.location.href = wpcomLink(
```

Comment:
I wonder, would it be better to put `wpcomLink` inside `getDomainAndPlanUpsellUrl`?